spec/ParseQuery.spec.js

```diff
@@ -1891,6 +1891,41 @@ describe('Parse.Query testing', () => {
     });
   });
 
+  it("dontSelect and equalTo (#3678)", function(done) {
```

Comment:
Thanks for your contribution, @xor22h.
Would you mind writing the actual test "name" or what we should expect this test to cover, instead of pointing to the issue? I think it makes it easier for readers.